### PR TITLE
[CRAW-138] Add number of new/recrawled URLs to mail report

### PR DESCRIPTION
**Pull request by vanchpuck**

### CODE REVIEW

File: src/java/org/apache/nutch/fetcher/FetcherJob.java [hunk 1]
```diff
@@ -108,6 +108,18 @@
     UNSPECIFIED_PAGE    
   }
   
+  public static enum PageRecency {
```
Comment (SergeyWeiss) [inline src/java/org/apache/nutch/fetcher/FetcherJob.java]: why don't we put such enums in separate files? can be a great help when merging changes from nutch

Comment (vanchpuck) [inline src/java/org/apache/nutch/fetcher/FetcherJob.java]: I propose to create separate task for that.

Comment (SergeyWeiss) [inline src/java/org/apache/nutch/fetcher/FetcherJob.java]: you do realise this task will live and die at the bottom of tasks lists? :)

**Comment by SergeyWeiss** [comment]
ok

**Pull request merged**